eslint.config.js

```diff
@@ -1,56 +1,42 @@
-import globals from 'globals';
-import tsPlugin from '@typescript-eslint/eslint-plugin';
-import tsParser from '@typescript-eslint/parser';
-import vuePlugin from 'vue-eslint-parser';
-import eslintJsPlugin from '@eslint/js';
-import stylistic from '@stylistic/eslint-plugin';
-
-const gqlSchemaPath = '././typeDefs/schema.graphql';
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+import stylistic from '@stylistic/eslint-plugin'
 
 export default [
-    // GLOBAL configuration
     {
+        // GLOBAL configuration
         ignores: [
             'dist/*',
-            'output/*',
+            '.output/*',
             '.nuxt/*',
             'coverage/*',
             'cypress/videos/*',
-            '.yarn/*'
+            '.yarn/*',
+            'typedefs/gqlTypes.ts'
         ],
     },
-    // Typescript and JS Linter combined (for all the main code files)
+    eslint.configs.recommended,
+    ...tseslint.configs.recommended,
     {
-        languageOptions: {
-            parser: vuePlugin.parser,
```

Comment:
I think this just removed the whole linting for vue files. We'll likely need to readd this back in